lib/puppet/parser/functions/validate_port.rb

```diff
@@ -18,6 +18,8 @@ module Puppet::Parser::Functions
     @return [Nil]
     EOS
 
+    function_simplib_deprecation(['validate_port', 'validate_port is deprecated, please use simplib::validate_port'])
```

Comment:
Hopefully, people won't be using these any longer, but it can still be useful if you're munging through Strings and don't want to write the `if/else` logic yourself.